### PR TITLE
Added floating text program

### PR DESCRIPTION
**Pull request by vchrombie**
Thank you for your contribution. Please provide the details requested below.

## ISSUE NUMBER

#41 
#20 

## SHORT DESCRIPTION

Floating Text is a simple program which displays a given input String as a floating string on the terminal.

## TESTING

`cd floating_text`
`python floating_text.py`

Added a [gif](https://github.com/vchrombie/Code-Sleep-Python/tree/master/floating_text/floating_text.gif) for your reference.

### CODE REVIEW

**Comment by prateekiiest** [comment]
please see the codacy PR reviews.

There's some issue on the os.system("cls")

**Comment by prateekiiest** [comment]
https://www.codacy.com/app/prateekkol21/Code-Sleep-Python/file/12710560219/issues/source?bid=5954332&fileBranchId=5974235#l181

**Comment by vchrombie** [comment]
Okay
I will fix that and send you a PR again.

Thank You.

**Comment by prateekiiest** [comment]
No need to send a PR again, You can edit that in this PR only

**Comment by vchrombie** [comment]
All the checks have passed!
Can you review my PR #89 please.

**Comment by vchrombie** [comment]
All the checks are passed.
Can you review my PR #89

File: floating_text/floating_text.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                       "*   *",
+                       "**** ",
+                       "*  * ",
+                       "*   *"  ],
+
+               "S" : [ " ****",
+                       "*    ",
+                       " *** ",
+                       "    *",
+                       "**** " ],
```
Comment (prateekiiest) [inline floating_text/floating_text.py]: What about numbers ?

Comment (prateekiiest) [reply floating_text/floating_text.py]: And remove the cls not found command.

Its obstructing the view

Comment (vchrombie) [reply floating_text/floating_text.py]: The program is just for English alphabets.

Comment (vchrombie) [reply floating_text/floating_text.py]: Okay, I will fix that.

**Comment by vchrombie** [comment]
The error in the gif is fixed.
Also the code is implemented for the numbers also.

Please review it again.

**Comment by vchrombie** [comment]
Can you review my PR #89 please.

**Pull request merged**